server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessagesMethodTest.java

```diff
@@ -220,6 +220,68 @@ public void getMessagesShouldReturnMessageWhenHtmlMessage() throws Exception {
     }
```

Comment:
you still miss the "content-type=text/plain with some html tags in the body " test case